buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -18,6 +18,9 @@ object Versions {
     // DI - Dagger
     const val dagger = "2.14.1"
 
+    //DI-KOIN
+    const val koin = "2.2.3"
```

Comment:
Can we migrate to[ KOIN V3](https://insert-koin.io/docs/setup/v3)?

Reply:
> Can we migrate to[ KOIN V3](https://insert-koin.io/docs/setup/v3)?

Yes. I'll bump it up.